doc/api/buffer.md

```diff
@@ -1025,6 +1025,9 @@ console.log(buf.toString('ascii'));
 * {ArrayBuffer} The underlying `ArrayBuffer` object based on
   which this `Buffer` object is created.
 
+This `ArrayBuffer` is not guaranteed to correspond exactly to the original
+`Buffer`. See the notes on `buf.byteOffset` for details.
```

Comment:
```suggestion
`Buffer`. See the notes on [`buf.byteOffset`][] for details.
```

Add the following at the bottom:
```markdown
[`buf.byteOffset`]: #buffer_buf_byteoffset
```

Reply:
I think a better approach is to say that the `Buffer` is a view over a *subset* of the `ArrayBuffer`. It *will* correspond to that subset only starting at the `byteOffset`.